### PR TITLE
Add 'Zero-to-deploy with Chef on GCP' tutorial to Community Tutorials

### PR DESCRIPTION
**Pull request by djmailhot**
Walks through a quick and minimal workflow to deploying resources on GCP with Chef via the [GCP cookbooks](https://supermarket.chef.io/cookbooks/google-cloud).

- [x] Tech writing / styleguide review - @steveperry-53    
- [ ] Formatting review for publishing - @jmdobry

### CODE REVIEW

File: tutorials/zero-to-deploy-with-chef.md [hunk 1]
```diff
@@ -0,0 +1,328 @@
+---
+title: Zero-to-Deploy with Chef on GCP
+description: Learn how to manage Google Compute Engine with Chef.
+author: djmailhot
+tags: Compute Engine, Chef
+date_published: 2018-03-28
+---
+
+This tutorial shows how to quickly setup infrastructure on Google Cloud Platform
```
Comment (steveperry-53) [inline tutorials/zero-to-deploy-with-chef.md]: setup -> set up

File: tutorials/zero-to-deploy-with-chef.md [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+
+## Setup
+
+### Provision a Compute Engine instance
+
+This tutorial is written using the **us-east1-b** Compute Engine zone. You may
+choose any zone.
+
+This tutorial is written using the **Ubuntu 16.04 LTS** machine image. You may
+use any machine image that that supports Chef. See the list of all [supported
```
Comment (steveperry-53) [inline tutorials/zero-to-deploy-with-chef.md]: that that -> that

File: tutorials/zero-to-deploy-with-chef.md [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+### Initialize a Chef repository
+
+On your `chef-workstation` instance:
+
+1.  Setup a `cookbooks` directory.
+
+        mkdir -p .chef/cookbooks
+        cd .chef
+
+1.  Configure git.
```
Comment (steveperry-53) [inline tutorials/zero-to-deploy-with-chef.md]: git -> Git
Occurs several times.

File: tutorials/zero-to-deploy-with-chef.md [hunk 4]
```diff
@@ -0,0 +1,328 @@
...
+### Provision a Compute Engine instance
+
+This tutorial is written using the **us-east1-b** Compute Engine zone. You may
+choose any zone.
+
+This tutorial is written using the **Ubuntu 16.04 LTS** machine image. You may
+use any machine image that that supports Chef. See the list of all [supported
+distributions for Chef Client](https://downloads.chef.io/chef).
+
+1.  In the Cloud Platform Console, go to the **Compute Engine >> [VM
```
Comment (steveperry-53) [inline tutorials/zero-to-deploy-with-chef.md]: Add "page" at the end of the sentence.

File: tutorials/zero-to-deploy-with-chef.md [hunk 5]
```diff
@@ -0,0 +1,328 @@
...
+    gcloud iam service-accounts create chef-service-account --display-name \
+    "chef service account"
+    gcloud iam service-accounts keys create ~/chef-account-key.json \
+    --iam-account chef-service-account@YOUR_PROJECT_NAME.iam.gserviceaccount.com
+
+In this case, the service account key will be downloaded as
+`~/chef-account-key.json`.
+
+Once your service account key is downloaded, you'll need to upload it to your
+new `chef-workstation` GCE instance:
```
Comment (steveperry-53) [inline tutorials/zero-to-deploy-with-chef.md]: Our branding guidelines don't allow GCE. Use "Compute Engine" instead.

File: tutorials/zero-to-deploy-with-chef.md [hunk 6]
```diff
@@ -0,0 +1,328 @@
...
+
+1.  Download the chef client package for **Ubuntu 16.04**.
+
+        wget https://packages.chef.io/files/stable/chef/13.8.5/ubuntu/16.04/chef_13.8.5-1_amd64.deb
+
+1.  Install it.
+
+        sudo dpkg -i chef_*
+
+NOTE: If you selected a different machine image for your GCE instance, you'll
```
Comment (steveperry-53) [inline tutorials/zero-to-deploy-with-chef.md]: GCE -> Compute Engine

File: tutorials/zero-to-deploy-with-chef.md [hunk 7]
```diff
@@ -0,0 +1,328 @@
...
+### Run Chef Client
+
+On `chef-workstation`, run `chef-client` in 'local mode' with your recipe:
+
+    chef-client --local-mode --runlist 'recipe[google-cloud::default]'
+
+You should see output streaming by as the command operates. It should terminate
+with something like `Chef Client finished, 2/8 resources updated in 35 seconds`.
+
+Awesomesauce! You just provisioned a Compute Engine instance on GCP using a
```
Comment (steveperry-53) [inline tutorials/zero-to-deploy-with-chef.md]: Avoid local expression when writing for an international audience.
Suggestion: Change Awesomesauce to Awesome.

**Comment by djmailhot** [comment]
Fixes submitted for all comments by steveperry-53

**Pull request merged**